modules/api/main.tf

```diff
@@ -7,6 +7,12 @@ locals {
     aws_ecs_service.api_ecs_service[0].cluster,
     "arn:aws:ecs:${var.region}:${var.account_id}:task-definition/nerves-hub-${terraform.workspace}-api:*",
   ]
+
+  ecs_shared_env_vars = <<EOF
+    { "name" : "ENVIRONMENT", "value" : "${terraform.workspace}" },
```

Comment:
There is a new variable you added `environment_name`, is it the same as `terraform.workspace`?  if so it would be good to be consistent and use one. 

Personally having a variable called `environment_name` is more consistent with our other apps and removes the implicit dependence on workspaces to name the environment. But I don't have strong feelings on that, just that we're consistent.

Reply:
@magreenbaum added a few more fixes in regards to this issue as the logging module had already been merged with `var.environment_name`